src/systems/filecoin_blockchain/struct/chain/chain.go

```diff
@@ -10,8 +10,8 @@ import (
 // For negative epochs, it should return a tipset composed of the genesis block.
 func (chain *Chain_I) TipsetAtEpoch(epoch abi.ChainEpoch) Tipset {
 	current := chain.HeadTipset()
-
-	for current.Epoch() > epoch && epoch >= genesisEpoch{
+	genesisEpoch := abi.ChainEpoch(5)
```

Comment:
`genesisEpoch` did not exist, so we have to declare it. I realize that setting it to `abi.ChainEpoch` is wrong, but couldn't find a reference to `genesisEpoch` anywhere.

Reply:
`0` would be a better value